src/lib/app.js

```diff
@@ -116,6 +116,8 @@ class App {
 
     router.get('/transfers/:id', transfers.getResource)
     router.get('/transfers/:id/state', transfers.getStateResource)
+    router.get('/transfers/:id/hash_preimage', transfers.getPreimage)
```

Comment:
The default behaviour for this endpoint is to return the preimage of the transfer's current state, unless the `condition_state` is specified as a query param, in which case we return the hash preimage for the specified state.